packages/material-tailwind-react/src/components/Tabs/TabPanel.tsx

```diff
@@ -54,7 +54,7 @@ export const TabPanel = React.forwardRef<HTMLDivElement, TabPanelProps>(
           className={tabPanelClasses}
           initial="unmount"
```

Comment:
The provided initial property cannot solve the problem. This is because the `unmount` animation is set after receiving the state